packages/eslint-plugin-react-hooks/src/RulesOfHooks.js

```diff
@@ -489,7 +489,7 @@ export default {
               // These are dangerous if you have inline requires enabled.
               const message =
                 `React Hook "${context.getSource(hook)}" cannot be called ` +
-                'at the top level. React Hooks must be called in a ' +
+                'outside of a React Function. React Hooks must be called in a ' +
```

Comment:
Note how this is inside `codePathNode.type === 'Program'`. So it looks like this message is correct but you may have found a legitimate bug. Could you add a test for a function that's not at the top level but produces this error message?

Also mind the extra message we have for any other function in the above branch.

Reply:
Thanks for the nits! 
I will look upto that soon!